fix: fix contrast issue and text color in dark mode

JustOndra opened the pull request:
I fixed the contrast issue by changing two css variables a added few classes to code. 
Now it should be readable.

![Snímek obrazovky 2024-08-30 v 23 27 10](https://github.com/user-attachments/assets/5550b412-3d19-45ba-bc9b-76dc5193d7ad)
![Snímek obrazovky 2024-08-30 v 23 27 37](https://github.com/user-attachments/assets/8d3c1cd5-75f0-4adb-9981-342dbc63e542)

guyb1 commented on src/pages/editor-page/canvas/table-node.tsx:
```diff
@@ -75,10 +75,12 @@ export const TableNode: React.FC<NodeProps<TableNodeType>> = ({
                 style={{ backgroundColor: table.color }}
             ></div>
-            <div className="flex items-center h-9 bg-slate-200 px-2 justify-between group">
+            <div className="flex items-center h-9 px-2 justify-between group">
```
this change also changes the color of the light mode table header, how about adding something like `dark:bg-...` instead?

guyb1 commented on src/pages/editor-page/canvas/toolbar/toolbar.tsx:
```diff
@@ -50,7 +50,7 @@ export const Toolbar: React.FC<ToolbarProps> = () => {
     return (
         <div className="px-1">
-            <Card className="shadow-none p-0 bg-secondary h-[44px]">
+            <Card className="shadow-none p-0 h-[44px]">
```
Same like the prev comment, it change also changes the color in light mode. I would use `dark:` prefix instead

guyb1 commented on src/globals.css:
```diff
@@ -3,67 +3,67 @@
+    .dark {
+        --background: 222.2 84% 4.9%;
+        --foreground: 210 40% 98%;
+        --card: 210 40% 98%;
+        --card-foreground: 222.2 84% 4.9%;
```
I think it's better not to change the predefined colors, only the use of them :)

guyb1 commented:
Hey @JustOndra ! thank you for your PR!
Left you few comments, basically, I think we should not change the colors defined in the globals.css, only the use them appropriate to the use case we need.
I wouldn't also change the colors in light mode, instead just add `dark:` prefix for the case of dark mode where it's needed.

Thank you for your help!

JustOndra commented:
@guyb1 Hey! Sorry for the delay. Life happened. :/

Reverted changes in globals.css and added few classes to html with dark: prefix.
If it's okay I can update the PR. :)

![Snímek obrazovky 2024-09-02 v 17 55 29](https://github.com/user-attachments/assets/d8f893db-e02c-470f-8778-9f100bf99595)
![Snímek obrazovky 2024-09-02 v 17 55 36](https://github.com/user-attachments/assets/6de2ad25-383f-4eaa-8bb9-b6a1038d73c1)

johnnyfish commented:
> @guyb1 Hey! Sorry for the delay. Life happened. :/
> 
> Reverted changes in globals.css and added few classes to html with dark: prefix. If it's okay I can update the PR. :)
> 
> ![Snímek obrazovky 2024-09-02 v 17 55 29](https://private-user-images.githubusercontent.com/56552571/363744510-d8f893db-e02c-470f-8778-9f100bf99595.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzNTk2NDUsIm5iZiI6MTcyNTM1OTM0NSwicGF0aCI6Ii81NjU1MjU3MS8zNjM3NDQ1MTAtZDhmODkzZGItZTAyYy00NzBmLTg3NzgtOWYxMDBiZjk5NTk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDEwMjkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4NTVmNmFiZDk0NDY1MmIyNDFiNmE5NjY3ZDM0YzllNDA4Y2Y1NTVlYWJhNmMxMGNlOThkNTg0ODFlNmMyYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._v6N28FX2XCHH7V4d6SrW4yw2_IZrtICI7dCuIyr2KA) ![Snímek obrazovky 2024-09-02 v 17 55 36](https://private-user-images.githubusercontent.com/56552571/363744512-6de2ad25-383f-4eaa-8bb9-b6a1038d73c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzNTk2NDUsIm5iZiI6MTcyNTM1OTM0NSwicGF0aCI6Ii81NjU1MjU3MS8zNjM3NDQ1MTItNmRlMmFkMjUtMzgzZi00ZWFhLThiYjktYjZhMTAzOGQ3M2MxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDEwMjkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkNjExMjZiM2Q5ODMwZGUxM2NmOTA4NTBlMjY4NDhkNWI3NzNiMzJhZjczMGQyMDUzMDdlMmU2M2FjNjQwNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AiCmJJbfNRtKWqtIZpXkh2i3t2s86c68eBTpIDOq9dc)

Thank you so much for your effort on this! I reviewed the changes, and I noticed that the color selection isn't fully aligned with our product styling.

Considering this, it might be best to close this PR for now. I’ll make a quick update to address these points. I truly appreciate the time and effort you’ve put into this. thank you again!

JustOndra commented:
Hey @johnnyfish ! I am sorry I didn't get notification that you closed the issue a PR.

It's okay, no problem. I am glad I could try it.
I will focus on the relationship issue now. :)
